compiler/util/files.cpp

```diff
@@ -690,6 +693,12 @@ void codegen_makefile(fileinfo* mainfile, const char** tmpbinname, bool skip_com
   // factor of 5 or so in time in running the test system, as opposed
   // to specifying BINNAME on the C compiler command line.
 
+  if (fLibraryCompile) {
+    fprintf(makefile.fptr, "HEADERNAME = %s%s\n\n", libmodeHeadername, ".h");
+    tmpheader = astr(tmpDirName, "/", strippedHeadername, ".h");
+    fprintf(makefile.fptr, "TMPHEADERNAME = %s\n", tmpheader);
```

Comment:
I'm not sure what the point of giving the header name to the Makefile is anyway.
Or putting it in the /tmp directory and then moving it.
Does this header need to be available when compiling the libsomething.a ?
If the answer is "yes", then this is OK. If the answer is "no", then maybe it should be simplified?

Reply:
I talked to Brad about this before doing it: there was a comment that it saved execution time to put everything in the temp directory first then move it out, so I'm following that recommendation and using the same treatment for the header file.